packages/invoices/src/OutboundShipment/DetailView/modals/ItemDetailsForm.tsx

```diff
@@ -14,19 +14,24 @@ import {
   Typography,
   InputLabel,
   ModalNumericInput,
+  FlatButton,
+  CheckIcon,
+  Radio,
+  RadioGroup,
+  FormControlLabel,
 } from '@openmsupply-client/common';
 import { ItemSearchInput } from '@openmsupply-client/system/src/Item';
 import { OutboundShipmentSummaryItem } from '../types';
+import { PackSizeController } from './ItemDetailsModal';
 
 interface ItemDetailsFormProps {
   allocatedQuantity: number;
-  quantity?: number;
   register: UseFormRegister<FieldValues>;
   summaryItem?: OutboundShipmentSummaryItem;
-  packSize: number;
   onChangeItem: (newItem: Item | null) => void;
-  onChangeQuantity: (quantity: number) => void;
-  setPackSize: (packSize: number) => void;
+  onChangeQuantity: (quantity: number, packSize: number | null) => void;
```

Comment:
calling back with the pack size selected and the quantity typed into the input. A null pack size is when we're using units.

This calls the `allocateQuantity` func which assigns a quantity to the batch rows with the right pack size

The 'quantity' only ended up being needed here so just shifted it to state here.